TaskEngine/Tasks/QueueAwakerTask.cs

```diff
@@ -134,7 +134,7 @@ private async Task PendingJobs()
                 //// Videos which haven't been converted to wav 
                 //(await context.Videos.Where(v => v.Medias.Any() && v.Audio == null).ToListAsync()).ForEach(v => _convertVideoToWavTask.Publish(v.Id));
                 // Videos which have failed in transcribing
-                (await context.Videos.Where(v => v.TranscribingAttempts < 3 && v.TranscriptionStatus != "NoError" && v.Medias.Any())
+                (await context.Videos.Where(v => v.TranscribingAttempts < 1 && v.TranscriptionStatus != "NoError" && v.Medias.Any())
```

Comment:
```suggestion
                (await context.Videos.Where(v => v.TranscribingAttempts < Globals.TRANSCRIBING_ATTEMPTS_FAILURE_THRESHOLD && v.TranscriptionStatus != "NoError" && v.Medias.Any())
```
Is this meant to be a configurable threshold? Would it make sense as a `Globals`?

Reply:
it is a cherry-picked commit that was a minimal change emergency bandaid that was directly applied to production that is now being backported into this PR. I agree it needs a better solution in the future